ros2topic/test/fixtures/controller_node.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from geometry_msgs.msg import TwistStamped
```

Comment:
Why using `geometry_msgs` for this test? Something from `std_msgs` would be a much lighter dependency.

Reply:
`ros2 topic delay` works with message types that have a header (with a stamp) and `std_msgs` doesn't have any. There are some in `visualization_msgs`, `diagnostic_msgs`, `stereo_msgs`, `sensor_msgs`, `trajectory_msgs` and `nav_msgs`. Would you rather switch to one of those? I don't have a strong opinion.